pkg/repository/olap/types.go

```diff
@@ -0,0 +1,29 @@
+package olap
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type WorkflowRun struct {
+	Id           uuid.UUID `json:"id"`
+	TaskId       int32     `json:"task_id"`
+	WorkerId     int32     `json:"worker_id"`
+	TenantId     uuid.UUID `json:"tenant_id"`
+	Status       string    `json:"status"`
+	Timestamp    time.Time `json:"timestamp"`
+	CreatedAt    time.Time `json:"created_at"`
+	RetryCount   int32     `json:"retry_count"`
+	ErrorMessage string    `json:"error_message"`
+}
+
+type Event struct {
```

Comment:
Let's add a field for additional data? We should at least maintain the following fields from the old `StepRunEvent` table:

```sql
CREATE TABLE "StepRunEvent" (
    "reason" "StepRunEventReason" NOT NULL,
    "severity" "StepRunEventSeverity" NOT NULL,
    "message" TEXT NOT NULL,
    "data" JSONB
);
```

I know there's some inconsistency with reason/severity/message in addition to the new status field. But not every event corresponds to a status change (like the `Requeueing (no worker available)` events). Up to you how you'd like to handle this.

Reply:
done, just added a naive `additional__` prefix + the table name for these, so e.g. `additional__step_run_event_reason` - figure something like that will be okay for now but lmk what you think!